smoke/LeafDevice/details/Details.cs

```diff
@@ -156,6 +159,34 @@ protected async Task VerifyDataOnIoTHub()
             await eventHubClient.CloseAsync();
         }
 
+        static Task<MethodResponse> DirectMethod(MethodRequest methodRequest, object userContext)
+        {
+            Console.WriteLine("Leaf device received direct method call...");
+            return Task.FromResult(new MethodResponse((int)HttpStatusCode.OK));
+        }
+
+        protected async Task VerifyDirectMethod()
+        {
+            //User Service SDK to invoke Direct Method on the device.
+            ServiceClient serviceClient =
+                ServiceClient.CreateFromConnectionString(this.context.IotHubConnectionString);
+
```

Comment:
Nit: Remove extra white lines..

Reply:
Done. 

---
In reply to: [219917970](https://github.com/Azure/iotedge/pull/344#discussion_r219917970) [](ancestors = 219917970)